src/RedisCache/RedisCache/help/New-AzRedisCache.md

```diff
@@ -78,6 +78,131 @@ PS C:\>New-AzRedisCache -ResourceGroupName "MyGroup" -Name "MyCache" -Location "
+    2. You will need SubnetID for VNET enable cache. Syntax of SubnetID is given below.
+ 
+You can create virtual network from powershell command [New-AzVirtualNetwork](https://docs.microsoft.com/en-us/powershell/module/az.network/new-azvirtualnetwork?view=azps-6.4.0). 
+    Syntax of SubnetID: /subscriptions/{subid}/resourceGroups/{resourceGroupName}/providers/Microsoft.ClassicNetwork/VirtualNetworks/{vnetName}/subnets/{subnetName}
+
+```
+PS C:\>New-AzRedisCache -ResourceGroupName "MyGroup" -Name "MyCache" -Location "Central US" -Size P1 -Sku "Premium" -SubnetId "/subscriptions/a559b6fd-3a84-40bb-a450-b0db5ed37dfe/resourceGroups/mygroup/providers/Microsoft.Network/virtualNetworks/MyNet/subnets/MySubnet"
+
+          PrimaryKey         : pJ+jruGKPHDKsEC8kmoybobH3TZx2njBR3ipEsquZFo=
+          SecondaryKey       : sJ+jruGKPHDKsEC8kmoybobH3TZx2njBR3ipEsquZFo=
+          ResourceGroupName  : MyGroup
+          Id                 : /subscriptions/a559b6fd-3a84-40bb-a450-b0db5ed37dfe/resourceGroups/mygroup/providers/Microsoft.Cache/Redis/MyCache
+          Location           : Central US
+          Name               : mycache
+          Type               : Microsoft.Cache/Redis
+          HostName           : mycache.redis.cache.windows.net
+          Port               : 6379
+          ProvisioningState  : creating
+          SslPort            : 6380
+          RedisConfiguration : {[maxmemory-policy, allkeys-random], [maxclients, 7500], [maxmemory-reserved, 200],
+                                [maxfragmentationmemory-reserved, 300]...} 
+          EnableNonSslPort   : False
+          RedisVersion       : 4.0.14
+          Size               : 6GB
+          Sku                : Premium
+          SubnetId           : /subscriptions/a559b6fd-3a84-40bb-a450-b0db5ed37dfe/resourceGroups/mygroup/providers/Microsoft.Network/virtualNetworks/MyNet/subnets/MySubnet
+          StaticIP           : 10.0.0.4  
+          Tag                : {}
+          Zone               : []
+```
+
+### Example 5: Configure data persistence for a Premium Azure Cache for Redis
+
+Persistence writes Redis data into an Azure Storage account that you own and manage. So before configuring data persistence you need to have [storage account] (https://docs.microsoft.com/en-us/azure/storage/common/storage-account-create?tabs=azure-powershell) in same resource group. Choose a storage account in the same region and subscription as the cache, and a Premium Storage account is recommended because premium storage has higher throughput.
+
+For setting up data persistence, there is need to specifies Redis Configuration settings.   
+For RDB back up enabled :-
+    - rdb-backup-enabled (Set true or false)
+    - rdb-storage-connection-string (Go to your **Storage account** on portal. In left coloumn select **Access keys**. Then click on **Show keys** and copy any **connection string** from there.)
```

Comment:
It is better to redirect to some storage account documentation for steps to get the connection string.
We can also add a command example to get the connection string.